### PR TITLE
chore(node): Update rexie and related wasm dependencies

### PR DESCRIPTION
**Pull request by fl0rek**
Upgrade rexie and related web dependencies and add required code changes.

### CODE REVIEW

File: tools/upgrade-deps.sh [hunk 1]
```diff
@@ -1,4 +1,9 @@
-#!/bin/bash
+#!/usr/bin/env bash
```
Comment (fl0rek) [inline tools/upgrade-deps.sh:1]: this should make this script more portable, `bash` isn't always in `/bin`

File: node/src/store/indexed_db_store.rs
Comment (oblique) [inline node/src/store/indexed_db_store.rs:1]: There are many changes for rexie. I prefer them in a separate PR.

Comment (fl0rek) [reply node/src/store/indexed_db_store.rs:1]: Reverted to just rexie and friends, will open separate one for generic deps upgrade

**Review by zvolin — APPROVED** [review]
nice

**Comment by zvolin** [comment]
@fl0rek I just noticed it's out of date and likely conflicts with update of libp2p

**Review by zvolin — APPROVED** [review]

**Pull request merged**